### PR TITLE
feat: enhance brew uninstall

### PR DESCRIPTION
**Pull request by Nutlope**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature

**What is the current behavior? (You can also link to an open issue here)**
brew uninstall only lists Formulae

**What is the new behavior (if this is a feature change)?**
brew uninstall lists Formulae + Casks, which we want.

**Additional info:**
- Addresses #371

### CODE REVIEW

File: dev/brew.ts [hunk 1]
```diff
@@ -135,15 +135,18 @@ export const completionSpec: Fig.Spec = {
...
-            return out.split("\n").map((formula) => {
-              return {
-                name: formula,
-                icon: "🍺",
-                description: "Installed formula",
-              };
-            });
+            return out
+              .split("\n")
+              .filter((ea) => !ea.includes("="))
```
Comment (QuiiBz) [inline dev/brew.ts:142]: Could you rename `ea` to a more verbose name?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Nice change! Just a variable name to update before merging this

**Comment by Nutlope** [comment]
> Nice change! Just a variable name to update before merging this

done!

**Review by QuiiBz — APPROVED** [review]
Thanks! Let's merge this cool enhancement!

**Pull request merged**